src/routes/Home/index.tsx

```diff
@@ -35,6 +36,7 @@ function Home(): ReactElement {
 
         <Card>
           <h2>Transactions to Sign</h2>
+          <TxsToConfirmList />
```

Comment:
In order to remain as modular as possible, I'd argue that the title belongs to the widget.

Reply:
Agree!